src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -571,6 +571,11 @@ private WebElement getIframe(String slotName, String src) {
     return driver.findElement(By.cssSelector("iframe[id*='" + src + "/" + slotName + "']"));
   }
 
+  public void waitForCSSPseudoElementsContentToMatchValue(final String cssSelector, final String elementName, final String value) {
```

Comment:
I expected a bit different thing here.
Some time ago we had in the repo the following method:
```java
protected String getPseudoElementValue(WebElement element, String pseudoElement, String cssValue) {
		JavascriptExecutor js = (JavascriptExecutor) driver;
		String value = (String) js.executeScript(
			"return getComputedStyle(arguments[0], arguments[1])[arguments[2]];",
			element, pseudoElement, cssValue
		);
		return value;
}
```